docs/notebook_validation.py

```diff
@@ -105,7 +105,7 @@ def print_results(success, failed, skipped=[]):
 
         ## `quantum_transformer`:
         ## See: https://github.com/NVIDIA/cuda-quantum/issues/2689
-        notebooks_skipped = ['quantum_transformer.ipynb']
+        notebooks_skipped = ['afqmc.ipynb', 'quantum_transformer.ipynb']
```

Comment:
Q: Is it intentional to mark this notebook as 'skipped' after pinning the numba version?